MyWeight/Screens/List/MassView.swift

```diff
@@ -68,10 +72,21 @@ public class MassView: UIView, ViewModelOwner {
 
         dateLabel.numberOfLines = 0
         dateLabel.textAlignment = .right
+        
+        contentView.addSubview(line)
+        
+        line.translatesAutoresizingMaskIntoConstraints = false
+        
+        line.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: grid*4).isActive = true
+        line.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -grid*4).isActive = true
+        line.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: 0).isActive = true
+       
+        line.widthAnchor.constraint(equalToConstant: 100).isActive = true
+        line.heightAnchor.constraint(equalToConstant: 1).isActive = true
 
         self.backgroundColor = style.backgroundColor
     }
-
+    
```

Comment:
This space is unnecessary, you can configure your Xcode to clean this spaces for you. Take a look [here](http://stackoverflow.com/questions/1390329/trim-trailing-spaces-in-xcode/11830067#11830067).